languages/javalang/builder-renderer/java-builder-client/src/main/kotlin/io/vrap/codegen/languages/javalang/client/builder/requests/JavaHttpRequestRenderer.kt

```diff
@@ -256,6 +299,15 @@ class JavaHttpRequestRenderer @Inject constructor(override val vrapTypeProvider:
             |public Map<String, String> getAdditionalQueryParams() {
             |   return this.additionalQueryParams;
             |}
+            |
+            |private String urlEncode(final String s){
+            |   try{
+            |        return URLEncoder.encode(s, "UTF-8");
+            |    }catch (UnsupportedEncodingException e) {
+            |        e.printStackTrace();
+            |        return null;
```

Comment:
If I get it right, this should never happen because `UTF-8` is supported. Then I would suggest to remove the print and replace it with an inline comment stating that this can never happen. WDYT?